optional-container-engine/test/config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   cwd: path.resolve(path.join(__dirname, `../`)),
   cmd: `node`,
   args: [`app.js`],
+  url: `http://localhost:8088`,
```

Comment:
I think the correct solution is to change this line to `port: 8088` and modify the repo tools to dynamically set the `PORT` environment variable for the each subprocess that gets spawned for each deployment. This way each app can default to `8080`, but during the tests each app will use a different port based on the `PORT` env var in its subprocess.